tests/rtt_ros2_params_tests/test/test_ros_params.cpp

```diff
@@ -0,0 +1,253 @@
+  ASSERT_TRUE(nullptr != local_params);
+
+
+  // Test member function
+  assign_properties(42, true);
+
+  // Check set non-existant parameters
+  ASSERT_TRUE(local_params->getOperation("setParameter"));
+  ASSERT_FALSE(local_params->getOperation("nonExistantOperation"));
+  RTT::OperationCaller<bool(std::string, rclcpp::ParameterValue)>
+  setparam_operation = local_params->getOperation("setParameter");
+  ASSERT_FALSE(
+    setparam_operation.call("fake_parameter", rclcpp::ParameterValue(42)));
+
+  // Check get non-existant parameters
+  ASSERT_TRUE(local_params->getOperation("getParameter"));
+  RTT::OperationCaller<rclcpp::ParameterValue(std::string)>
+  getparam_operation = local_params->getOperation("getParameter");
+  ASSERT_EQ(
+    rclcpp::PARAMETER_NOT_SET,
+    getparam_operation.call("fake_parameter").get_type()
+  );
+
+  // Check setting a parameter (INTEGER)
+  rtt_ros2_node::getNode(this)->declare_parameter("int_parameter");
+  ASSERT_TRUE(
+    setparam_operation.call("int_parameter", rclcpp::ParameterValue(41)));
+
+  // Check getting a parameter (INTEGER)
+  rclcpp::ParameterValue new_int = getparam_operation.call("int_parameter");
+  ASSERT_EQ(rclcpp::PARAMETER_INTEGER, new_int.get_type());
+  ASSERT_EQ(new_int.get<int>(), 41);
+
+  // Check setting a parameter (BOOL)
+  rtt_ros2_node::getNode(this)->declare_parameter("bool_parameter");
+  ASSERT_TRUE(
+    setparam_operation.call("bool_parameter", rclcpp::ParameterValue(false)));
+
+  // Check getting a parameter (BOOL)
+  rclcpp::ParameterValue new_bool = getparam_operation.call("bool_parameter");
+  ASSERT_EQ(rclcpp::PARAMETER_BOOL, new_bool.get_type());
+  ASSERT_EQ(new_bool.get<bool>(), false);
+
+  // Check storing a parameter provoke failure
+  ASSERT_TRUE(local_params->getOperation("storeProperty"));
+  RTT::OperationCaller<bool(std::string, std::string)>
+  storeprop_operation = local_params->getOperation("storeProperty");
+  ASSERT_FALSE(storeprop_operation.call("fake_property", "int_parameter"));
+  ASSERT_FALSE(storeprop_operation.call("int_property", "fake_parameter"));
+
+  // Check storing a parameter (INTEGER)
+  ASSERT_EQ(getparam_operation.call("int_parameter").get<int>(), 41);
+  ASSERT_TRUE(storeprop_operation.call("int_property", "int_parameter"));
+  ASSERT_EQ(getparam_operation.call("int_parameter").get<int>(), 42);
+
+  // Check storing a parameter (BOOL)
+  ASSERT_EQ(getparam_operation.call("bool_parameter").get<bool>(), false);
+  ASSERT_TRUE(storeprop_operation.call("bool_property", "bool_parameter"));
+  ASSERT_EQ(getparam_operation.call("bool_parameter").get<bool>(), true);
+
+  // Check loading a parameter common
+  ASSERT_TRUE(local_params->getOperation("loadProperty"));
+  RTT::OperationCaller<bool(std::string, std::string)>
+  loadprop_operation = local_params->getOperation("loadProperty");
+  ASSERT_FALSE(loadprop_operation.call("fake_property", "fake_parameter"));
+  ASSERT_TRUE(loadprop_operation.call("fake_property", "int_parameter"));
```

Comment:
It is expected to be true `EXPECT_TRUE`, because the policy I implemented is that unknown properties are created as orphan properties that is allocated in the service `Params`.
I check then, the value.